docs/src/components/PlaygroundCodeEditor.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from '../pages/playground.module.css';
+import { ChangeEvent, useState } from 'react';
+
+interface PlaygroundCodeEditorProps {
+  onChange: (content: object | Error) => void;
+  initialCodeEditorContent: string;
+}
+
+function PlaygroundCodeEditor({
+  onChange,
+  initialCodeEditorContent,
+}: PlaygroundCodeEditorProps): JSX.Element {
+  const [currCodeEditorContent, setCurrCodeEditorContent] =
```

Comment:
```suggestion
  const [content, setContent] =
```
this is internal you don't need to be super descriptive with names that are not part of the public APi

Reply:
Also curr ti is wierd abreviation. I assume it stands for current? Avoid shortcuts like this if they are not obvious.